benchmarks/benchmark_functions/default_settings.py

```diff
@@ -86,9 +86,10 @@ def _vqe_defaults(hyperparams):
 	hf_state = qchem.hf_state(electrons, qubits)
 	ansatz = partial(UCCSD, init_state=hf_state, s_wires=s_wires, d_wires=d_wires)
 	params = np.array([3.14545258, 3.13766988, -0.21446816])
+	Hamiltonian = qml.Hamiltonian(H_coeffs, H_ops)
 
-	Hamiltonian = hyperparams.pop('Hamiltonian', qml.Hamiltonian(H_coeffs, H_ops))
-	ansatz = hyperparams.pop('template', ansatz)
+	Hamiltonian = hyperparams.pop('Hamiltonian', Hamiltonian)
```

Comment:
Wait, isn't the `Hamiltonian` above overwritten?

Reply:
The default Hamiltonian is the simple one for hydrogen defined in this file. In the heavy benchmark, a new Hamiltonian for LiH is defined, which replaces the default one.

Reply:
The current philosophy is that all the default molecular parameters are those of hydrogen. If we remove the hydrogen Hamiltonian from default_setting.py, then we need to have a dummy Hamiltonian since vqe.py gets the parameters through default_setting.py as: `ham, ansatz, params, n_steps, device, options_dict = _vqe_defaults(hyperparams)`.